pkg/collector/corechecks/embed/process/process_agent.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build process && (darwin || freebsd)
```

Comment:
Out of curiosity, I though `freebsd` was not officially supported. Do we have other `freebsd` flags we could get rid of?

Reply:
We have some freebsd build tags here and there, we considered removing them a couple months but were told to keep them
We're not testing or building the agent for freebsd as of now but just in case we're keeping the existing build tags (when it makes sense)
I think the agent hasn't been built on freebsd for years so it would most likely not compile 😬